CMakeLists.txt

```diff
@@ -459,7 +459,6 @@ if (MFC_DOCUMENTATION)
         BUILD_COMMAND     ""
         INSTALL_COMMAND   ""
     )
-
```

Comment:
Did you intend to include this change?

Reply:
@henryleberre, no. There should be no (intended) changes to this file other than the ones I followed from you